### PR TITLE
Bulgarian Language added

### PR DESCRIPTION
**Pull request by ddachkinov**

### CODE REVIEW

**Comment by phelmig** [comment]
Hi @ddachkinov,
Thank you for your translation. Since we can't validate the contribution's quality we are currently requiring all translations to be reviewed by at least two native speakers (we count you as the contributor as the first). If everything is fine, a simple comment on this pull request is sufficient. If you can reach out to your personal network to find someone for a peer review it would be highly appreciated.

**Comment by nstankovicc** [comment]
Native Bulgarian here. I am picking up review.

**Comment by Nikoogle** [comment]
Native Bulgarian here. I will gladly review.

File: site/bg.njk [hunk 1]
```diff
@@ -0,0 +1,145 @@
+---
+layout: default
+meta_title: кампания за спирането на пандемията COVID-19 
+meta_description: Нашите правителства не могат да се справят без нашата помощ, трябва да предприемем действия сега! Изпълнете своята роля и останете у дома мамка му!
+body_class: home
+title: '#StayTheFuckHome'
+lang: en
```
Comment (Nikoogle) [inline site/bg.njk]: Needs to be:
`bg`

File: site/bg.njk [hunk 2]
```diff
@@ -0,0 +1,145 @@
+---
+layout: default
+meta_title: кампания за спирането на пандемията COVID-19 
+meta_description: Нашите правителства не могат да се справят без нашата помощ, трябва да предприемем действия сега! Изпълнете своята роля и останете у дома мамка му!
```
Comment (Nikoogle) [inline site/bg.njk:4]: Edit suggested, to avoid being wordy.

Нашите правителства не могат да се справят без нашата помощ, трябва да предприемем действия сега! Изпълнете своята роля и **си стойте вкъщи, мамка му!**

File: site/bg.njk [hunk 3]
```diff
@@ -0,0 +1,145 @@
+---
+layout: default
+meta_title: кампания за спирането на пандемията COVID-19
```
Comment (Nikoogle) [inline site/bg.njk]: Capital Letters./Главни букви.
**К**ампания...

File: site/bg.njk [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+        <h1 class="page-title">{{ title }}</h1>
+        <h2 class="page-subtitle">Кампания за спирането на пандемията COVID-19</h2>
+
+        <div class="content">
+
+            <p class="text-xl">
+                Нашите правителства прилагат мерки за контрол на разпространението на
+                <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a>
+                вируса и разрастване на <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019">COVID-19 пандемията</a>.
```
Comment (Nikoogle) [inline site/bg.njk]: Let's use the Bulgarian version of the article -- [Here](https://bg.wikipedia.org/wiki/%D0%9A%D0%BE%D1%80%D0%BE%D0%BD%D0%B0%D0%B2%D0%B8%D1%80%D1%83%D1%81%D0%BD%D0%B0_%D0%B1%D0%BE%D0%BB%D0%B5%D1%81%D1%82_2019)

File: site/bg.njk [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+    <section id="container-centre" class="column centre flex-1">
+
+        <h1 class="page-title">{{ title }}</h1>
+        <h2 class="page-subtitle">Кампания за спирането на пандемията COVID-19</h2>
+
+        <div class="content">
+
+            <p class="text-xl">
+                Нашите правителства прилагат мерки за контрол на разпространението на
+                <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a>
```
Comment (Nikoogle) [inline site/bg.njk]: Let's use the BG version of the article -- [Here](https://bg.wikipedia.org/wiki/SARS-CoV-2)

File: site/bg.njk [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+                вируса и разрастване на <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019">COVID-19 пандемията</a>.
+                Нереагирането навреме, успокояването на масата и стремежът им да стабилизират икономиката ги възпират да предприемат
```
Comment (Nikoogle) [inline site/bg.njk]: Word Choice.

Липсата на реакция, вместо Нереагирането Навреме.

File: site/bg.njk [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+
+        <div class="content">
+
+            <p class="text-xl">
+                Нашите правителства прилагат мерки за контрол на разпространението на
+                <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a>
+                вируса и разрастване на <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019">COVID-19 пандемията</a>.
+                Нереагирането навреме, успокояването на масата и стремежът им да стабилизират икономиката ги възпират да предприемат 
+                необходимите мерки за защита на милиони от тази болест. Това обаче далеч не е задача само на правителствата.
+                Време е ние, като граждани на тази Земя, да предприемем действия сега и да дадем своята роля в борбата с COVID-19.
```
Comment (Nikoogle) [inline site/bg.njk]: Word Choice.

Да дадем своя принос, вместо да дадем своята роля.

File: site/bg.njk [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+                <br /><br />
+                Да си го кажем директно: <strong>Стойте си у дома мамка му!</strong>
```
Comment (Nikoogle) [inline site/bg.njk]: Missing comma.

Стойте си у дома**,**мамка му!

File: site/bg.njk [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+            </p>
+
+            <h2 class="text-3xl">Манифест за самокарантирането</h2>
+
+            <p>
+                Без конкретно установено лечение и без ваксина достъпни до поне една година, основен начин за намаляване на ефекта от
+                коронавирус пандемията е да се понижат неговите шансове за разпространение. Следният списък с действия, подредени от
```
Comment (Nikoogle) [inline site/bg.njk]: Word Choice.

...неговите шансове за разпространение --> Шансовете на вируса за разпространение.

File: site/bg.njk [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+                Най-лесните за прилагане към най-ефективните съвете в борбата срещу пандемията. Той трябва да служи като набор от насоки за хората, които желаят да се присъединят
```
Comment (Nikoogle) [inline site/bg.njk]: Typo + Comma.
Най-лесните за прилагане към най-ефективните съвете в борбата срещу пандемията. --> Най-лесните за прилагане, към най-ефективните съвет**И**, в борбата срещу пандемията.

File: site/bg.njk [hunk 11]
```diff
@@ -0,0 +1,145 @@
...
+                къв кампанията и предприемат действия, от които ще имат огромно значение.
```
Comment (Nikoogle) [inline site/bg.njk]: Typo + Comma + Word Choice.
 
къ**М** кампанията, и предприемат действия, които са истински значими.

File: site/bg.njk [hunk 12]
```diff
@@ -0,0 +1,145 @@
...
+            </p>
+
+            <ol class="text-xl mb-12 list-decimal pl-12">
+                <li class="pb-2">Не изпадайте в паника, но бъдете нащрек.</li>
+                <li class="pb-2">Мийте си ръцете често и спазвайте етикет при кашляне и кихане (Използвайте лакът).</li>
```
Comment (Nikoogle) [inline site/bg.njk]: Adding Context.
(Използвайте лакът) --> (Използвайте лактите си, а не ръце).

File: site/bg.njk [hunk 13]
```diff
@@ -0,0 +1,145 @@
...
+                <li class="pb-2">Докосвайти лицето си възможно най-малко, включително устата, носът и очите си.</li>
```
Comment (Nikoogle) [inline site/bg.njk]: Typo.
ДокосвайтИ -> Докосвайт**е**

File: site/bg.njk [hunk 14]
```diff
@@ -0,0 +1,145 @@
...
+                <li class="pb-2">Практикувайте социално дистанциране, без прегръдки и целувки, без ръкостискания, без High 5 (гепи). Ако трябва, използвайте по-безопасни алтернативи. </li>
```
Comment (Nikoogle) [inline site/bg.njk]: High 5 -> (Забиване/Даване на 5).

Let's translate this fully, w/o doing a bg-eng mix.

File: site/bg.njk [hunk 15]
```diff
@@ -0,0 +1,145 @@
...
+                <li class="pb-2">Не посещавайте концерти, театрални постановки, спортни събития или други масови развлекателни събития. </li>
+                <li class="pb-2">Въздържайте се от посещение на музеи, изложби, киносалони, нощни клубове и други места за забавление. </li>
+                <li class="pb-2">Стойте далеч от социални събития и събития, като срещи в клуба, религиозни служби и частни партита. </li>
```
Comment (Nikoogle) [inline site/bg.njk]: Missing punctuation, that creates logical separation between parts of the sentence.

Стойте далеч от социални събития**,**  и събития,

File: site/bg.njk [hunk 16]
```diff
@@ -0,0 +1,145 @@
...
+            <h2 class = "text-3xl"> Защо е толкова важно </h2>
+            <P>
+                SARS-CoV-2 е силно инфекциозен и потенциално смъртоносен вирус, който причинява респираторно заболяване, наречено COVID-19.
+                Може да го познаете под едно от многото му имена, включително 2019-nCoV, нов коронавирус, коронавирус Ухан, корана или грип Ухан, или просто коронавирус.
+                Всички те се отнасят до един и същ вирус, който тази кампания има за цел да спре.
+            </P>
+
+            <P>
+                През последните месеци и седмици вирусът и заболяването от него се сравняват отново и отново с грипния вирус и сезонния грип.
+                Поради близките симптоми и привидно сходните усложнения, съвсем естествено е хората да ги бъркат
```
Comment (Nikoogle) [inline site/bg.njk]: I think we should add a comma after "бъркат." And start the next sentence on a new line. It's too wordy.

File: site/bg.njk [hunk 17]
```diff
@@ -0,0 +1,145 @@
...
+                    </p>
+                </li>
+                <li class="pb-2">
+                    <h3>Без лечение, без ваксина, без имунитет</h3>
+                    <p class="font-normal">
+                        Тъй като SARS-CoV-2 се появи едва наскоро, няма конкретно лечение срещу COVID-19. Необходими са повече изследвания, за да се лекуват заразените хора ефективно.
+                        По същия начин, все още няма и ваксина за SARS-CoV-2,<sup><a class="source-link" href="https://www.cdc.gov/coronavirus/2019-ncov/about/prevention-treatment.html" target="_blank">[6]</a></sup>
+                        а разработването на такава ще отнеме значително време. Без ваксини и без развит имунитет, всеки е податлив на инфекция.
+                        Докато при повечето заразени хора ще отмине само с леки симптоми, липсата на имунитет у хората може да доведе до тежки заболявания при значителен брой рискови индивиди - нашите най-обичани родители, баби и дядовци.
```
Comment (Nikoogle) [inline site/bg.njk]: Missing clarification, going out of context.

Докато при повечето заразени хора **инфекцията** ще отмине...

File: site/bg.njk [hunk 18]
```diff
@@ -0,0 +1,145 @@
...
+                    </p>
+                </li>
+                <li class="pb-2">
+                    <h3>Експоненциален растеж</h3>
+                    <p class="font-normal">
+                        Поради липсата на имунитет към новия вирус, всеки човек на земята може да се зарази. И въпреки, че експоненциалният растеж може да изглежда бавен,
+                        може да доведе до невероятно високи числа за доста кратък период от време. С удвоилият се само за няколко дни брой на заразените хора<sup><a class="source-link" href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/" target="_blank">[7]</a></sup>,
```
Comment (Nikoogle) [inline site/bg.njk]: Missing linking word.

...**Той**(растежа) може...

File: site/bg.njk [hunk 19]
```diff
@@ -0,0 +1,145 @@
...
+                        нашите медицински системи потенциално ще се претоварят, което ще доведе до по-голям брой смъртни случаи от хора, които не не могат да получат необходимите медицински грижи.
+                    </p>
+                </li>
+            </ol>
+
+            <p class="text-2xl">
+                Присъединявайки се към кампанията и следователно ограничавате възможностите за нови инфекции, вие не само защитавате себе си, но и помагате за ограничаване на разпространението за всички останали,
```
Comment (Nikoogle) [inline site/bg.njk]: ...и следователно **ограничавайки**...

File: site/bg.njk [hunk 20]
```diff
@@ -0,0 +1,145 @@
...
+                особено тези, които са изложени на по-висок риск - нашите най-обичани родители, баби и дядовци, да понесат тежки последици от този вирус.
+                <br /><br />
+                
+                <strong>Ако смятате тази кампания зя полезна, моля, споделете я в социалната мрежа, по имейл или просто кажете на приятелите и семейството си за нея. И моля ви, #StayTheFuckHome.</strong>
+            </p>
+
+            <div class="flex justify-center pb-12">
+                <a href="https://twitter.com/share?ref_src=twsrc%5Etfw" class="twitter-share-button" data-size="large" data-text="I will do my part in the fight against #COVID19. I urge you to do the same!" data-url="https://staythefuckhome.com" data-hashtags="StayTheFuckHome" data-show-count="false">Tweet</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (Nikoogle) [inline site/bg.njk]: I will do my part in the fight against #COVID19. I urge you to do the same!

"Аз ще направя всичко необходимо, за да преборим #COVID19. Приветствам Ви да направите същото!

File: site/bg.njk [hunk 21]
```diff
@@ -0,0 +1,145 @@
...
+                <iframe class="ml-6" src="https://www.facebook.com/plugins/share_button.php?href=https%3A%2F%2Fstaythefuckhome.com&layout=button_count&size=large&appId=764658637392408&width=111&height=28" width="111" height="28" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowTransparency="true" allow="encrypted-media"></iframe>
+            </div>
+
+            <p class="text-sm text-gray-500 mb-10 mt-10">
+                За всеки, който иска да допринесе за #StayTheFuckHome уебсайта и кампанията, да даде своя принос. Може да се намери в GitHub на: <a href="https://github.com/flore2003/staythefuckhome" target="_blank">https://github.com/flore2003/staythefuckhome</a>
+                <br />
+                Ако имате някакви други данни относно съдържанието на уебсайта, моля, пишете ни на <a href="mailto:contribute@staythefuckhome.com">contribute@staythefuckhome.com</a>.
+                В момента получаваме много имейли, така че може да не се свържем веднага с вас.
```
Comment (Nikoogle) [inline site/bg.njk]: ...с вас веднага...

**Review by Nikoogle — CHANGES_REQUESTED** [review]
I am kindly requesting changes to this branch.
I have forwarded the same for review by fellow Bulgarian Native Speakers.

Thank you for your contribution. It's a great effort 👍

**Comment by dachkinova** [comment]
As a native speaker, I would say that the translation is accurate and complete

File: site/bg.njk (hunk 16 above)
Comment (ddachkinov) [reply site/bg.njk]: reworded it differently, to be less wordy

File: site/bg.njk (hunk 21 above)
Comment (ddachkinov) [reply site/bg.njk]: Thank you for your time and suggestions, I've made the corrections and I'm submitting the code. 
Stay safe!

**Comment by ddachkinov** [comment]
Thank you for your time and suggestions, I've made the corrections and I've committed the file.
Stay safe everyone!

**Review by Nikoogle — APPROVED** [review]
Thank you, looks good!

**Comment by Nikoogle** [comment]
@phelmig.

This has been approved by two reviewers.
Can you please check it out?

**Comment by ddachkinov** [comment]
Can you check it out @phelmig, it has been 3 months now, it would have been nice for some folks to read it.
